subsys/bluetooth/host/uuid.c

```diff
@@ -130,3 +132,49 @@ void bt_uuid_to_str(const struct bt_uuid *uuid, char *str, size_t len)
 		return;
 	}
 }
+
+int bt_uuid_from_str(const char *str, struct bt_uuid *uuid)
```

Comment:
Seems a bit inconsistent with the `bt_uuid_create()` API return type. Also, there doesn't really seem to exist any other category of failure than "I can't interpret this as a UUID".